### ISSUE TITLE
Get the latest lblrtm/lnfl

### ISSUE THREAD

**Issue by shihyuntang** [kgullikson88/Telluric-Fitter#39]
Hi @gully and @kgullikson88 ,

I was trying to see if one can use the latest lblrtm/lnfl from [ARE-RC's github](https://github.com/AER-RC/LBLRTM) instead of the current old? version that telfit download form a zenodo space.
And... it turns out to be super easy!
Just clone both the lblrtm/lnfl from ARE-RC's github (and follow their setup instruction) to your `.Telfit` dir, overwrite the one lblrtm/lnfl for sure, re-run `python setup.py build/install`. AND done!

BTW, this will also fix the error pop up when you try to use the latest ifort as fortran complier.
(Which... using ifort seems to still be the only way to fix this issue #9)

If you think we should updated this for telfit, i will make a pull requisition when i have time.
Cheers!

**Comment by kgullikson88** [comment]
Yeah, I think it makes sense to include the latest release from their github. The reason it used the zenodo link previously was just because lblrtm was not _on_ github at the time TelFit was written.

**Comment by shihyuntang** [comment]
Great!
But, i am not sure what will be the best way to implement the installation of pkg from other git repo.
Also, this section of code in setup.py [line 190--192](https://github.com/kgullikson88/Telluric-Fitter/blob/0690886548c02953a0856cdea8556c39f94f9e20/setup.py#L190) need to match the latest line list version (which is v3.8 now).
Not sure how to streamline these...
